src/OmniSharp.Roslyn.CSharp/Workers/Formatting/FormattingWorker.cs

```diff
@@ -104,8 +100,8 @@ public static async Task<IEnumerable<LinePositionSpanTextChange>> GetFormattedTe
         private static async Task<Document> FormatDocument(Document document, OmniSharpOptions omnisharpOptions, ILoggerFactory loggerFactory, TextSpan? textSpan = null)
         {
             var optionSet = omnisharpOptions.FormattingOptions.EnableEditorConfigSupport
-                ? await document.Project.Solution.Workspace.Options.WithEditorConfigOptions(document.FilePath, loggerFactory)
-                : document.Project.Solution.Workspace.Options;
+                ? await document.GetOptionsAsync()
```

Comment:
oh I see, so this will inlcude `EditorConfigDocumentOptionsProviderFactory` in the options computation?

Reply:
Right, since I opened the issue, we have moved the OptionsProvider registration into the base Workspace constructor.